pom.xml

```diff
@@ -1214,6 +1214,7 @@
 
 				<module>akamai</module>
 				<module>aws</module>
+				<module>batch</module>
```

Comment:
De (integration) tests gaan nu nog fout omdat de module nog niet in de full-bundle zit. Zou je die daar als dependency toe kunnen voegen?

Reply:
Ah, dat had ik effe niet door. Ik krijg geen mailtjes trouwens als de 2e build bv start en als die faalt. Dus ik was hiervan niet bewust.